### PR TITLE
audit: simplify command-line spelling check

### PR DESCRIPTION
**Pull request by afh**

### CODE REVIEW

File: Library/Homebrew/cmd/audit.rb [hunk 1]
```diff
@@ -360,12 +360,8 @@ def audit_desc
...
     end
 
-    if desc =~ %r[[Cc]ommandline]
-      problem "It should be \"command-line\", not \"commandline\"."
-    end
-
-    if desc =~ %r[[Cc]ommand line]
-      problem "It should be \"command-line\", not \"command line\"."
+    if desc =~ %r[([Cc]ommand ?line)]
+      problem "It should be \"command-line\", not \"#{$1}\""
```
Comment (xu-cheng) [inline Library/Homebrew/cmd/audit.rb]: I know it's the existing logic to be blamed. But maybe `"It should be \"command-line\", not \"#{$1}\" in description"` is better, so we know such problem comes from description.

**Comment by afh** [comment]
How about `"Description should use \"#{$2}ommand-line instead of \"#{$1}\""`?
`$2` would be a new regex group within the first matching `[Cc]`.

**Comment by xu-cheng** [comment]
Should be `"Description should use \"command-line\" instead of \"#{$1}\""`.

**Comment by bfontaine** [comment]
Please rebase this on top of master.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/9e679750e54516576ea4e1e6823c1e2a43ceca58. Thanks @afh!

**Pull request closed without merging**